__tests__/rules/naming-convention.test.ts

```diff
@@ -9,6 +9,7 @@ ruleTester.run('naming-convention', namingConvention, {
     { code: '', filename: 'kebab-case.js' },
     { code: '', filename: 'index.js', options: [{ rule: 'PascalCase' }] },
     { code: '', filename: 'index.js', options: [{ rule: '[a-z]*' }] },
+    { code: '', filename: '.eslintrc.js', options: [{ excepts: ['\..+'] }] },
```

Comment:
Looking at this line, I thought you might just want to remove the file from the target of rule/naming-convention.

If so, you can override the rules for the specific file in eslintrc.
```javascript
module.exports = {
  ...
  overrides: [
    {
      files: ['.eslintrc.js'],
      plugins: ['filenames-simple'],
      rules: {
        'filenames-simple/naming-convention': 'off'
      }
    }
  ]
};
```